services/121-service/src/utils/send-xlsx-response.spec.ts

```diff
@@ -0,0 +1,48 @@
+import * as XLSX from 'xlsx';
+
+import { arrayToXlsx } from '@121-service/src/utils/send-xlsx-response';
+
+describe('arrayToXlsx', () => {
+  it('should convert array to XLSX buffer', () => {
+    // Arrange
+    const testData = [
+      { name: 'John', age: 30, city: 'New York' },
+      { name: 'Jane', age: 25, city: 'Boston' },
+      { name: 'Bob', age: 35, city: 'Chicago' },
+    ];
+
+    // Act
+    const result = arrayToXlsx(testData);
+
+    // Assert
+    expect(result).toBeInstanceOf(Buffer);
+    expect(result.length).toBeGreaterThan(0);
+    const workbook = XLSX.read(result, { type: 'buffer' });
+    expect(workbook.SheetNames).toEqual(['data']);
+    const worksheet = workbook.Sheets['data'];
```

Comment:
I don't think this file should be part of this commit. I think something went wrong with rebasing